### PR TITLE
feat(jtk): migrate deferred commands to artifact projection

### LINKED ISSUE #206: 5/8 feat(jtk): migrate deferred commands to the default artifact contract
## What

Migrate the remaining high-value `jtk` commands that were deferred from the first artifact-projection slice because of `--full` flag conflicts.

This issue should move those commands to the default artifact contract once the CLI surface has been made unambiguous.

## Why

`jtk` is only partially migrated today. The first slice established the projection pattern for safe command families, but several important workflows are still on the old raw/near-raw JSON path because their local `--full` semantics conflict with the new artifact model.

Finishing those deferred migrations is necessary for `jtk` to present a consistent agent-default contract across its core workflows and to fully retire the legacy compact path later.

## Notes

- This issue depends on the `--full` collision work being resolved first.
- The focus is the deferred command set rather than introducing new projection architecture.

### PR DESCRIPTION
**Pull request by rianjs**
## Summary

Closes #206

- Migrate `issues get/list/search`, `comments list`, and `automation get` to artifact projection
- Add `CommentArtifact` with truncated body in agent mode (~200 chars) for comment triage
- Add `AutomationRuleArtifact` with component summary ("3 total - 1 trigger(s), 2 action(s)")
- Expand `IssueArtifact` full-mode fields: created, updated, reporter, labels
- JSON output now uses standard artifact format: `{"results": [...], "_meta": {"count": N, "hasMore": bool}}`

This completes the jtk artifact migration started in #199 and unblocked by #205.

## Test plan

- [x] `make build` passes
- [x] `make test` passes
- [x] `make lint` passes for jtk
- [ ] Manual: `jtk issues get PROJ-123 -o json` returns IssueArtifact
- [ ] Manual: `jtk issues get PROJ-123 -o json --full` includes created/updated/reporter/labels
- [ ] Manual: `jtk issues list -p PROJ -o json` returns artifact list format
- [ ] Manual: `jtk comments list PROJ-123 -o json` returns CommentArtifact list
- [ ] Manual: `jtk auto get <UUID> -o json` returns AutomationRuleArtifact

### CODE REVIEW

File: tools/jtk/internal/artifact/comment.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+// ProjectComment projects an api.Comment to a CommentArtifact.
+func ProjectComment(c *api.Comment, mode artifact.Type) *CommentArtifact {
+	body := ""
+	if c.Body != nil {
+		body = strings.TrimSpace(c.Body.ToPlainText())
+	}
+
+	// In agent mode, truncate body for triage readability
+	if !mode.IsFull() && len(body) > 200 {
+		body = body[:200] + "..."
```
Comment (monit-reviewer) [inline tools/jtk/internal/artifact/comment.go]: 🟡 **Medium** (harness-engineering:architecture-reviewer): Body is truncated at byte offset 200 (`body[:200]`), not at a rune boundary. For comments containing multi-byte UTF-8 characters (emoji, CJK, accented Latin), this silently produces invalid UTF-8, which JSON marshaling will mangle (replacing with U+FFFD). Use `[]rune(body)[:200]` or a rune-aware truncation helper instead.

File: tools/jtk/internal/artifact/automation.go [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+			a.Tags = rule.Tags
+		}
+	}
+
+	return a
+}
+
+// summarizeComponents creates a compact summary of rule components.
+// Format: "3 total — 1 trigger(s), 2 action(s)"
+func summarizeComponents(components []api.RuleComponent) string {
```
Comment (monit-reviewer) [inline tools/jtk/internal/artifact/automation.go:50]: 🔵 **Low** (harness-engineering:architecture-reviewer): `summarizeComponents` uses `len(components)` for the total count but only enumerates TRIGGER, CONDITION, and ACTION in the breakdown. Any component with an unrecognized `Component` value is silently dropped from the parts string but still inflates the total, producing misleading output like '4 total — 1 trigger(s), 1 action(s)' when there are 2 unknown-type components. Consider adding an 'other' bucket or surfacing unrecognized types.

File: tools/jtk/internal/cmd/comments/comments.go [hunk 3]
```diff
@@ -66,8 +70,11 @@ func runList(ctx context.Context, opts *root.Options, issueKey string, maxResult
 		return nil
 	}
 
-	if opts.Output == "json" {
-		return v.JSON(result.Comments)
+	if v.Format == view.FormatJSON {
```
Comment (monit-reviewer) [inline tools/jtk/internal/cmd/comments/comments.go:73]: 🔵 **Low** (harness-engineering:enforcement-reviewer): `hasMore` is computed as `result.StartAt + len(result.Comments) < result.Total`. If `result.Total` is 0 (a known Jira Cloud edge case for certain filters), `hasMore` will always be false even when additional pages exist, silently truncating results. Consider guarding against `result.Total == 0`.

File: tools/jtk/internal/artifact/automation_test.go [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+		testutil.Equal(t, len(labels), 1)
+	})
+}
+
+func TestSummarizeComponents(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name       string
+		components []api.RuleComponent
```
Comment (monit-reviewer) [inline tools/jtk/internal/artifact/automation_test.go:198]: 🔵 **Low** (harness-engineering:architecture-reviewer): `TestSummarizeComponents` is table-driven but does not use `t.Run()` subtests. When a case fails, the output only identifies the outer test, not which case failed. All other table-driven tests in the file use `t.Run()`. Align for diagnosability.

File: tools/jtk/internal/cmd/issues/search_fields_test.go [hunk 5]
```diff
@@ -400,10 +410,10 @@ func TestRunList_AutoPaginationJSON(t *testing.T) {
 	err = runList(context.Background(), opts, "TEST", "", 120, "", false, "")
```
Comment (monit-reviewer) [inline tools/jtk/internal/cmd/issues/search_fields_test.go:410]: 🔵 **Low** (harness-engineering:knowledge-reviewer): `TestRunList_AutoPaginationJSON` was updated to assert `count` and `hasMore` but drops the result-content assertions present in the analogous search test (e.g., `result.Results[0].Key`). The list path has weaker content coverage than search for the same migration.

**Review by monit-reviewer — CHANGES_REQUESTED** [review]
## Automated PR Review



**Reviewed commit:** `897ca34`

### Summary

| Reviewer | Findings |
|----------|----------|
| harness-engineering:architecture-reviewer | 3 |
| harness-engineering:enforcement-reviewer | 1 |
| harness-engineering:knowledge-reviewer | 1 |

<details>
<summary><strong>harness-engineering:architecture-reviewer (3 findings)</strong></summary>

**⚠️ Should Fix** - `tools/jtk/internal/artifact/comment.go:33`
> Body is truncated at byte offset 200 (`body[:200]`), not at a rune boundary. For comments containing multi-byte UTF-8 characters (emoji, CJK, accented Latin), this silently produces invalid UTF-8, which JSON marshaling will mangle (replacing with U+FFFD). Use `[]rune(body)[:200]` or a rune-aware truncation helper instead.

**💡 Suggestion** - `tools/jtk/internal/artifact/automation.go:50`
> `summarizeComponents` uses `len(components)` for the total count but only enumerates TRIGGER, CONDITION, and ACTION in the breakdown. Any component with an unrecognized `Component` value is silently dropped from the parts string but still inflates the total, producing misleading output like '4 total — 1 trigger(s), 1 action(s)' when there are 2 unknown-type components. Consider adding an 'other' bucket or surfacing unrecognized types.

**💡 Suggestion** - `tools/jtk/internal/artifact/automation_test.go:198`
> `TestSummarizeComponents` is table-driven but does not use `t.Run()` subtests. When a case fails, the output only identifies the outer test, not which case failed. All other table-driven tests in the file use `t.Run()`. Align for diagnosability.

</details>

<details>
<summary><strong>harness-engineering:enforcement-reviewer (1 findings)</strong></summary>

**💡 Suggestion** - `tools/jtk/internal/cmd/comments/comments.go:73`
> `hasMore` is computed as `result.StartAt + len(result.Comments) < result.Total`. If `result.Total` is 0 (a known Jira Cloud edge case for certain filters), `hasMore` will always be false even when additional pages exist, silently truncating results. Consider guarding against `result.Total == 0`.

</details>

<details>
<summary><strong>harness-engineering:knowledge-reviewer (1 findings)</strong></summary>

**💡 Suggestion** - `tools/jtk/internal/cmd/issues/search_fields_test.go:410`
> `TestRunList_AutoPaginationJSON` was updated to assert `count` and `hasMore` but drops the result-content assertions present in the analogous search test (e.g., `result.Results[0].Key`). The list path has weaker content coverage than search for the same migration.

</details>

*3 info-level observations excluded. Run with `--verbose` to include.*

---
<details>
<summary>Completed in 2m 01s | $0.81</summary>

| Field | Value |
|---|---|
| Reviewers | hybrid-synthesis, database:sql-reviewer, security:code-auditor, harness-engineering:knowledge-reviewer, harness-engineering:enforcement-reviewer, harness-engineering:architecture-reviewer, harness-engineering:legibility-reviewer |
| Engine | claude |
| Reviewed by | [pr-review-daemon](https://github.com/SignalFT/pr-review-daemon) · [monit-pr-reviewer](https://github.com/apps/monit-pr-reviewer) |
| Duration | 2m 01s wall · 5m 09s compute (Reviewers: 1m 21s · Synthesis: 37s) |
| Cost | $0.81 |
| Tokens | 249.3k in / 25.8k out |
| Turns | 14 |

</details>

File: tools/jtk/internal/artifact/comment.go (hunk 1 above)
Comment (rianjs) [reply tools/jtk/internal/artifact/comment.go]: Fixed - now using rune-based truncation to avoid cutting multi-byte UTF-8 characters.

File: tools/jtk/internal/artifact/automation.go (hunk 2 above)
Comment (rianjs) [reply tools/jtk/internal/artifact/automation.go:50]: Fixed - added 'other' bucket for unknown component types.

File: tools/jtk/internal/cmd/comments/comments.go (hunk 3 above)
Comment (rianjs) [reply tools/jtk/internal/cmd/comments/comments.go:73]: Fixed - added guard for Total==0 edge case. Falls back to checking if we received a full page of results.

File: tools/jtk/internal/artifact/automation_test.go (hunk 4 above)
Comment (rianjs) [reply tools/jtk/internal/artifact/automation_test.go:198]: Fixed - now using t.Run() subtests for diagnosability. Also added test case for unknown component types.

File: tools/jtk/internal/cmd/issues/search_fields_test.go (hunk 5 above)
Comment (rianjs) [reply tools/jtk/internal/cmd/issues/search_fields_test.go:410]: Fixed - added key assertions for content coverage parity with search test.

File: tools/jtk/internal/artifact/issue.go [hunk 6]
```diff
@@ -19,8 +16,12 @@ type IssueArtifact struct {
 	Assignee string `json:"assignee,omitempty"`
```
Comment (monit-reviewer) [inline tools/jtk/internal/artifact/issue.go:16]: 🟡 **Medium** (harness-engineering:enforcement-reviewer): `IssueArtifact` has no `Description` field in either agent or full mode, despite description being the primary content of a Jira issue. Every other artifact in this PR includes a primary content field (`CommentArtifact.Body`, `AutomationRuleArtifact.Description`). Previously `jtk issues get -o json` returned the full API object including the description body. Omitting it makes `issues get -o json --full` a silent regression for callers reading issue descriptions. A `Description` field (full-mode only, ADF-to-plain-text) should be added for parity.

File: tools/jtk/internal/cmd/comments/comments.go [hunk 7]
```diff
@@ -66,8 +70,19 @@ func runList(ctx context.Context, opts *root.Options, issueKey string, maxResult
...
+		arts := jtkartifact.ProjectComments(result.Comments, opts.ArtifactMode())
+		// Use authoritative pagination metadata from API response.
+		// Guard against Total==0 edge case in Jira Cloud by also checking
+		// if we received a full page of results.
+		hasMore := false
```
Comment (monit-reviewer) [inline tools/jtk/internal/cmd/comments/comments.go:78]: 🔵 **Low** (harness-engineering:enforcement-reviewer): The `Total==0` hasMore fallback (`len(result.Comments) == maxResults`) introduces a false-positive risk: if an issue has exactly `maxResults` comments and the API legitimately returns `Total==0`, `hasMore` will be `true` incorrectly, causing callers to make a spurious extra page request that returns empty results. The comment acknowledges the heuristic, but there is no test covering this edge case to guard against regression if the logic is changed later.

File: tools/jtk/internal/artifact/comment_test.go [hunk 8]
```diff
@@ -0,0 +1,187 @@
...
+
+	art := ProjectComment(comment, artifact.Agent)
+
+	// Body should be truncated to 200 chars + "..."
+	testutil.Equal(t, len(art.Body), 203)
+	testutil.True(t, strings.HasSuffix(art.Body, "..."))
+}
+
+func TestProjectComment_FullMode(t *testing.T) {
+	t.Parallel()
```
Comment (monit-reviewer) [inline tools/jtk/internal/artifact/comment_test.go:83]: 🔵 **Low** (harness-engineering:enforcement-reviewer): The rune-based truncation fix is only tested with ASCII input (`strings.Repeat("A", 300)`). The implementation explicitly uses `[]rune` to avoid cutting multi-byte UTF-8 sequences, but no test exercises truncation at a multi-byte character boundary (e.g., CJK or emoji) to verify the rune-aware logic actually fires correctly.

File: tools/jtk/internal/cmd/automation/get.go [hunk 9]
```diff
@@ -50,8 +53,8 @@ func runGet(ctx context.Context, opts *root.Options, ruleID string, showComponen
 		return err
 	}
 
-	if opts.Output == "json" {
-		return v.JSON(rule)
+	if v.Format == view.FormatJSON {
```
Comment (monit-reviewer) [inline tools/jtk/internal/cmd/automation/get.go:56]: 🔵 **Low** (harness-engineering:architecture-reviewer): There is no integration-level test for the new `automation get` JSON path, analogous to the `TestRunSearch_AutoPaginationJSON` and `TestRunList_AutoPaginationJSON` tests added/updated for issues. The artifact projection unit tests cover `ProjectAutomationRule` in isolation, but `runGet`'s new `RenderArtifact` call path is not exercised end-to-end.

File: tools/jtk/internal/cmd/issues/get.go [hunk 10]
```diff
@@ -45,9 +46,9 @@ func runGet(ctx context.Context, opts *root.Options, issueKey string, noTruncate
 		return err
 	}
 
-	// For JSON output, return the full issue
-	if opts.Output == "json" {
-		return v.JSON(issue)
+	// For JSON output, return the projected artifact
```
Comment (monit-reviewer) [inline tools/jtk/internal/cmd/issues/get.go:49]: 🔵 **Low** (harness-engineering:architecture-reviewer): Breaking JSON schema change across multiple commands: `issues get/list/search`, `comments list`, and `auto get` now return artifact projection format instead of raw Jira API payloads. Existing consumers relying on fields like `fields.description`, `fields.issuetype`, `Issues[]`, or `Pagination.IsLast` will silently receive an incompatible schema. The commit uses `feat:` without a `BREAKING CHANGE:` footer; downstream consumers have no version signal that the shape changed.

**Review by monit-reviewer — CHANGES_REQUESTED** [review]
## Automated PR Review



**Reviewed commit:** `269ebb5` | **Previous:** `897ca34` (incremental)

### Summary

| Reviewer | Findings |
|----------|----------|
| harness-engineering:architecture-reviewer | 2 |
| harness-engineering:enforcement-reviewer | 3 |

<details>
<summary><strong>harness-engineering:architecture-reviewer (2 findings)</strong></summary>

**💡 Suggestion** - `tools/jtk/internal/cmd/automation/get.go:56`
> There is no integration-level test for the new `automation get` JSON path, analogous to the `TestRunSearch_AutoPaginationJSON` and `TestRunList_AutoPaginationJSON` tests added/updated for issues. The artifact projection unit tests cover `ProjectAutomationRule` in isolation, but `runGet`'s new `RenderArtifact` call path is not exercised end-to-end.

**💡 Suggestion** - `tools/jtk/internal/cmd/issues/get.go:49`
> Breaking JSON schema change across multiple commands: `issues get/list/search`, `comments list`, and `auto get` now return artifact projection format instead of raw Jira API payloads. Existing consumers relying on fields like `fields.description`, `fields.issuetype`, `Issues[]`, or `Pagination.IsLast` will silently receive an incompatible schema. The commit uses `feat:` without a `BREAKING CHANGE:` footer; downstream consumers have no version signal that the shape changed.

</details>

<details>
<summary><strong>harness-engineering:enforcement-reviewer (3 findings)</strong></summary>

**⚠️ Should Fix** - `tools/jtk/internal/artifact/issue.go:16`
> `IssueArtifact` has no `Description` field in either agent or full mode, despite description being the primary content of a Jira issue. Every other artifact in this PR includes a primary content field (`CommentArtifact.Body`, `AutomationRuleArtifact.Description`). Previously `jtk issues get -o json` returned the full API object including the description body. Omitting it makes `issues get -o json --full` a silent regression for callers reading issue descriptions. A `Description` field (full-mode only, ADF-to-plain-text) should be added for parity.

**💡 Suggestion** - `tools/jtk/internal/cmd/comments/comments.go:78`
> The `Total==0` hasMore fallback (`len(result.Comments) == maxResults`) introduces a false-positive risk: if an issue has exactly `maxResults` comments and the API legitimately returns `Total==0`, `hasMore` will be `true` incorrectly, causing callers to make a spurious extra page request that returns empty results. The comment acknowledges the heuristic, but there is no test covering this edge case to guard against regression if the logic is changed later.

**💡 Suggestion** - `tools/jtk/internal/artifact/comment_test.go:56`
> The rune-based truncation fix is only tested with ASCII input (`strings.Repeat("A", 300)`). The implementation explicitly uses `[]rune` to avoid cutting multi-byte UTF-8 sequences, but no test exercises truncation at a multi-byte character boundary (e.g., CJK or emoji) to verify the rune-aware logic actually fires correctly.

</details>

*2 info-level observations excluded. Run with `--verbose` to include.*

*6 PR discussion threads considered.*

---
<details>
<summary>Completed in 3m 24s | $1.22</summary>

| Field | Value |
|---|---|
| Mode | Re-review · Cycle 2 · Session resumed |
| Reviewers | hybrid-synthesis, database:sql-reviewer, security:code-auditor, harness-engineering:knowledge-reviewer, harness-engineering:enforcement-reviewer, harness-engineering:architecture-reviewer, harness-engineering:legibility-reviewer |
| Engine | claude |
| Reviewed by | [pr-review-daemon](https://github.com/SignalFT/pr-review-daemon) · [monit-pr-reviewer](https://github.com/apps/monit-pr-reviewer) |
| Duration | 3m 24s wall · 7m 46s compute (Reviewers: 1m 43s · Synthesis: 1m 32s) |
| Cost | $1.22 |
| Tokens | 311.8k in / 28.3k out |
| Turns | 14 |

</details>

File: tools/jtk/internal/artifact/issue.go (hunk 6 above)
Comment (rianjs) [reply tools/jtk/internal/artifact/issue.go:16]: Fixed - added Description field to IssueArtifact (full-mode only) for parity with other artifacts.

File: tools/jtk/internal/artifact/comment_test.go (hunk 8 above)
Comment (rianjs) [reply tools/jtk/internal/artifact/comment_test.go:83]: Added multi-byte character truncation test using CJK characters to verify rune-aware logic.

File: tools/jtk/internal/cmd/comments/comments.go (hunk 7 above)
Comment (rianjs) [reply tools/jtk/internal/cmd/comments/comments.go:78]: Acknowledged - the Total==0 fallback is defensive; adding a test for this edge case is low priority given it's a rare Jira Cloud quirk. The current logic errs on the side of attempting pagination rather than missing results.

File: tools/jtk/internal/cmd/automation/get.go (hunk 9 above)
Comment (rianjs) [reply tools/jtk/internal/cmd/automation/get.go:56]: Acknowledged - integration test coverage for automation get is lower than issues. The artifact projection unit tests provide core coverage; integration tests can be added incrementally.

File: tools/jtk/internal/cmd/issues/get.go (hunk 10 above)
Comment (rianjs) [reply tools/jtk/internal/cmd/issues/get.go:49]: This is an intentional breaking change documented in the PR description. The artifact contract was introduced specifically to standardize JSON output across commands. The previous raw API output was undocumented behavior. This migration completes the artifact projection work started in #199/#204.

File: tools/jtk/internal/artifact/comment.go [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+	"github.com/open-cli-collective/jira-ticket-cli/api"
+)
+
+// CommentArtifact is the projected output for a comment.
+// Body is included in agent mode (truncated) because it's the primary semantic content.
+type CommentArtifact struct {
+	// Agent fields - body is primary content, truncated for triage
+	ID      string `json:"id"`
+	Author  string `json:"author"`
+	Created string `json:"created"`
```
Comment (monit-reviewer) [inline tools/jtk/internal/artifact/comment.go:17]: 🔵 **Low** (harness-engineering:knowledge-reviewer): `CommentArtifact.Body` lacks the `omitempty` JSON tag. When a comment body is empty, it serializes as `"body": ""` rather than omitting the field. Every other optional string field in both `CommentArtifact` and `IssueArtifact` uses `omitempty`. If intentional (body is the primary content and should always appear), a comment explaining the omission would prevent future contributors from adding it inadvertently.

File: tools/jtk/internal/artifact/issue_test.go [hunk 12]
```diff
@@ -65,6 +80,12 @@ func TestProjectIssue_FullMode(t *testing.T) {
 	// Full-only fields populated
```
Comment (monit-reviewer) [inline tools/jtk/internal/artifact/issue_test.go:80]: 🔵 **Low** (harness-engineering:knowledge-reviewer): `IssueArtifact` lacks a JSON serialization test comparable to `TestProjectAutomationRule_JSONSerialization` and `TestProjectComment_JSONSerialization` added in this PR. The five new full-mode fields (`created`, `updated`, `reporter`, `labels`, `description`) are verified via struct-field assertions, but there is no test confirming that `description` and `labels` are absent from agent-mode JSON output via `omitempty`, nor that all five appear in full-mode serialized JSON. This gap is especially relevant for `description`, which is conditionally set — a future refactor removing the guard could silently include it in agent mode.

**Review by monit-reviewer — APPROVED** [review]
## Automated PR Review



**Reviewed commit:** `7410426` | **Previous:** `269ebb5` (incremental)

> **Approved** with 2 non-blocking suggestions below. Address at your discretion.

### Summary

| Reviewer | Findings |
|----------|----------|
| harness-engineering:knowledge-reviewer | 2 |

<details>
<summary><strong>harness-engineering:knowledge-reviewer (2 findings)</strong></summary>

**💡 Suggestion** - `tools/jtk/internal/artifact/comment.go:17`
> `CommentArtifact.Body` lacks the `omitempty` JSON tag. When a comment body is empty, it serializes as `"body": ""` rather than omitting the field. Every other optional string field in both `CommentArtifact` and `IssueArtifact` uses `omitempty`. If intentional (body is the primary content and should always appear), a comment explaining the omission would prevent future contributors from adding it inadvertently.

**💡 Suggestion** - `tools/jtk/internal/artifact/issue_test.go:80`
> `IssueArtifact` lacks a JSON serialization test comparable to `TestProjectAutomationRule_JSONSerialization` and `TestProjectComment_JSONSerialization` added in this PR. The five new full-mode fields (`created`, `updated`, `reporter`, `labels`, `description`) are verified via struct-field assertions, but there is no test confirming that `description` and `labels` are absent from agent-mode JSON output via `omitempty`, nor that all five appear in full-mode serialized JSON. This gap is especially relevant for `description`, which is conditionally set — a future refactor removing the guard could silently include it in agent mode.

</details>

*3 info-level observations excluded. Run with `--verbose` to include.*

*10 PR discussion threads considered.*

---
<details>
<summary>Completed in 3m 20s | $1.36</summary>

| Field | Value |
|---|---|
| Mode | Re-review · Cycle 3 · Session resumed |
| Reviewers | hybrid-synthesis, database:sql-reviewer, security:code-auditor, harness-engineering:knowledge-reviewer, harness-engineering:enforcement-reviewer, harness-engineering:architecture-reviewer, harness-engineering:legibility-reviewer |
| Engine | claude |
| Reviewed by | [pr-review-daemon](https://github.com/SignalFT/pr-review-daemon) · [monit-pr-reviewer](https://github.com/apps/monit-pr-reviewer) |
| Duration | 3m 20s wall · 8m 26s compute (Reviewers: 1m 49s · Synthesis: 1m 11s) |
| Cost | $1.36 |
| Tokens | 353.7k in / 31.2k out |
| Turns | 14 |

</details>
